public/app/features/search/page/components/FolderView.test.tsx

```diff
@@ -122,6 +122,24 @@ describe('FolderView', () => {
     expect(await screen.findByRole('button', { name: 'General' })).toBeInTheDocument();
   });
 
+  it('sorts folders alphabetically by name by default', async () => {
+    render(
+      <FolderView
+        hidePseudoFolders
+        onTagSelected={mockOnTagSelected}
+        selection={mockSelection}
+        selectionToggle={mockSelectionToggle}
+      />
+    );
+    const testIdPrefix = selectors.components.Search.folderHeader('');
+    const names = (
+      (await screen.findAllByTestId(testIdPrefix, { exact: false })).map((el) =>
+        el.dataset.testid?.replace(testIdPrefix, '')
+      ) as string[]
+    ).sort();
+    expect(names).toEqual(['Another folder', 'General', 'My folder 1']);
```

Comment:
@ArturWierzbicki i noticed that the test output sorts `General` differently than when viewed in the browser. is the sorting being modified by a parent of this component or something?

Reply:
yeah the sorting is done in the backend, and `General` is added as the first item:

https://github.com/grafana/grafana/blob/6920f4347fc194aa926a38b637dc2c312dd2b476/public/app/features/search/page/components/FolderView.tsx#L47-L61

Reply:
👍 removed the test